source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -398,10 +496,11 @@ A Connection MUST NOT be checked out until it is readyToUse. In addition, the Po
         else if totalConnectionCount < maxPoolSize:
           connection = create connection
         # If there is no background thread, the pool MUST ensure that
-        # at least minPoolSize connections are in the pool.
-        # This SHOULD be done in a non-blocking manner
+        # there are at least minPoolSize total connections.
```

Comment:
Same as previous comment, either use "available + in use" instead of "total" or remove the requirement completely.